config/initializers/octokit.rb

```diff
@@ -1,10 +1,14 @@
 Octokit.middleware = Faraday::RackBuilder.new do |builder|
-  builder.use :http_cache, store: Rails.cache, logger: Rails.logger, shared_cache: false, serializer: Marshal
+  logger = Logger.new(nil)
+  builder.use :http_cache, store: Rails.cache, logger: logger, shared_cache: false, serializer: Marshal
   builder.use Octokit::Middleware::FollowRedirects
   builder.use Octokit::Response::RaiseError
   builder.use Octokit::Response::FeedParser
   builder.use FaradayMiddleware::Gzip
   builder.use :instrumentation
   builder.request :retry
   builder.adapter :typhoeus
+  # ethon doesn't expose any other way to shut up its logging :(
```

Comment:
Haha yikes. Might be at least a little more explanitory to call it nil_logger or somthin?